### ISSUE TITLE
.btn-close background SVG is not rendering correctly

### ISSUE THREAD

**Issue by colincalnan** [coreui/coreui#293]
There appears to be an issue with the SVG formatting for the Close button on things like modals and alert.

The React documentation appears to have a working version of the button - https://coreui.io/react/docs/4.0/components/modal/
The Vanilla JS version and the current npm version if not working - https://coreui.io/docs/4.0/components/modal/

I think this may be something to do with the value of `$btn_close_bg` in _variables.scss - https://github.com/coreui/coreui/blob/f92b2b3cd3082caf318b0c979bc4cef1c42da3cb/scss/_variables.scss#L1934

Or the use of `escape-svg` in the `_close.scss` file - https://github.com/coreui/coreui/blob/8a00c7c881475ead6ad6f37beb232e38b13200c7/scss/_close.scss#L12

I have added the following to my custom css for now to get it to work.

```
.btn-close {
   background: transparent var(--cui-btn-close-bg,url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' fill='rgba(44, 56, 74, 0.95)' viewBox='0 0 16 16'%3E%3Cpath d='M.293.293a1 1 0 0 1 1.414 0L8 6.586 14.293.293a1 1 0 1 1 1.414 1.414L9.414 8l6.293 6.293a1 1 0 0 1-1.414 1.414L8 9.414l-6.293 6.293a1 1 0 0 1-1.414-1.414L6.586 8 .293 1.707a1 1 0 0 1 0-1.414z'/%3E%3C/svg%3E")) center/1em auto no-repeat
}
```

**Comment by wittfabian** [comment]
You can set the value in coreui/scss/_variables.scss:

$btn-close-bg:               
url("data:image/svg+xml;charset=utf-8,<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16' fill='rgba(44, 56, 74, .95)'><path d='M.293.293a1 1 0 0 1 1.414 0L8 6.586 14.293.293a1 1 0 1 1 1.414 1.414L9.414 8l6.293 6.293a1 1 0 0 1-1.414 1.414L8 9.414l-6.293 6.293a1 1 0 0 1-1.414-1.414L6.586 8 .293 1.707a1 1 0 0 1 0-1.414z'/></svg>") !default;

In this case, the fill value is static, not a variable.

**Comment by mrholek** [comment]
Fixed in v4.1
